### PR TITLE
add Captcha setting for volto-redmine-helpdesk addon

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

**Comment by nileshgulia1** [comment]
Frontend counterpart https://github.com/eea/volto-eea-kitkat/pull/85

File: setup.py [hunk 1]
```diff
@@ -52,6 +52,7 @@
             'eea.volto.policy',
             'pas.plugins.oidc',
             'collective.taxonomy',
+            'collective.volto.subsites==1.3.1',
```
Comment (avoinea) [inline setup.py:55]: This pin is not good for Plone 6 usage

**Review by avoinea — CHANGES_REQUESTED** [review]

File: eea/kitkat/upgrades/configure.zcml [hunk 2]
```diff
@@ -63,4 +63,16 @@
 
   </genericsetup:upgradeSteps>
 
+  <genericsetup:upgradeSteps
+    source="4.0"
+    destination="4.2"
```
Comment (avoinea) [inline eea/kitkat/upgrades/configure.zcml:68]: @Petchesi-Iulian Destination should be 5.0 if you bumped version.txt to 5.0

File: eea/kitkat/profiles/default/metadata.xml [hunk 3]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-  <version>4.0</version>
+  <version>4.2</version>
```
Comment (avoinea) [inline eea/kitkat/profiles/default/metadata.xml:3]: 5.0

File: eea/kitkat/profiles/default/metadata.xml [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
   <dependencies>
     <dependency>profile-collective.taxonomy:default</dependency>
+    <dependency>profile-collective.volto.subsites:default</dependency>
```
Comment (avoinea) [inline eea/kitkat/profiles/default/metadata.xml:6]: To be removed.

**Review by avoinea — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (nileshgulia1) [reply setup.py]: Done in [c7b653a](https://github.com/eea/eea.kitkat/pull/19/commits/c7b653a42b6a0237af1e27f0a48578ea19e39afa)

File: eea/kitkat/profiles/default/metadata.xml (hunk 4 above)
Comment (nileshgulia1) [reply eea/kitkat/profiles/default/metadata.xml]: done [c7b653a](https://github.com/eea/eea.kitkat/pull/19/commits/c7b653a42b6a0237af1e27f0a48578ea19e39afa)

File: eea/kitkat/configure.zcml [hunk 5]
```diff
@@ -6,6 +6,7 @@
   <i18n:registerTranslations directory="locales" />
 
   <include package="collective.taxonomy" />
+  <include package="collective.volto.subsites" />
```
[the remark this answers is not in the document]
Comment (nileshgulia1) [reply eea/kitkat/configure.zcml]: Done [c7b653a](https://github.com/eea/eea.kitkat/pull/19/commits/c7b653a42b6a0237af1e27f0a48578ea19e39afa)

**Review by avoinea — APPROVED** [review]

**Pull request merged**